tests/test_blocks.py

```diff
@@ -12,27 +16,96 @@ def setUpTestData(cls):
         fake_file = ContentFile("Test")
         fake_file.name = "test.mp3"
 
-        cls.media = Media.objects.create(
-            title="Test", duration=1000, file=fake_file, type="audio"
+        cls.audio = Media.objects.create(
+            title="Test audio", duration=1000, file=fake_file, type="audio"
+        )
+
+        fake_file = ContentFile("Test")
+        fake_file.name = "test.mp4"
+
+        cls.video = Media.objects.create(
+            title="Test video", duration=1024, file=fake_file, type="video"
         )
 
     def test_abstract_render_raises_not_implemented_error(self):
         block = AbstractMediaChooserBlock()
         with self.assertRaises(NotImplementedError):
-            block.render(self.media)
+            block.render(self.audio)
 
     def test_render_calls_render_basic(self):
         class TestMediaChooserBlock(AbstractMediaChooserBlock):
             def render_basic(self, value, context=None):
                 return value.file.name
 
         block = TestMediaChooserBlock()
-        self.assertEqual(block.render(self.media), "media/test.mp3")
+        self.assertEqual(block.render(self.audio), "media/test.mp3")
 
     def test_media_block_get_form_state(self):
         block = AbstractMediaChooserBlock()
-        form_state = block.get_form_state(self.media.id)
-        self.assertEqual(self.media.id, form_state["id"])
-        self.assertEqual(self.media.title, form_state["title"])
-        edit_link = reverse("wagtailmedia:edit", args=(self.media.id,))
+        form_state = block.get_form_state(self.audio.id)
+        self.assertEqual(self.audio.id, form_state["id"])
+        self.assertEqual(self.audio.title, form_state["title"])
+        edit_link = reverse("wagtailmedia:edit", args=(self.audio.id,))
         self.assertEqual(edit_link, form_state["edit_link"])
+
+    def test_abstract_media_block_queryset(self):
+        block = AbstractMediaChooserBlock()
+
+        # note: can't use self.assertQuerysetEqual() as it is funky in Django < 3.2
+        self.assertListEqual(
+            list(block.field.queryset.order_by("pk").values_list("pk", flat=True)),
+            list(Media.objects.order_by("pk").values_list("pk", flat=True)),
+        )
+
+        block = AbstractMediaChooserBlock(media_type="audio")
+        self.assertListEqual(
+            list(block.field.queryset.values_list("pk", flat=True)),
```

Comment:
This assertion is missing the `order_by`, does this matter?

Also applies to:
- ``test_audio_chooser_block_field_queryset``
- ``test_video_chooser_block_field_queryset``

Reply:
in this case it doesn't matter as both return 1 item. it fails for > 1 results